src/crimson/common/logfile.cc

```diff
@@ -0,0 +1,76 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#include <fstream>
+#include <seastar/util/log.hh>
+
+#include "common/debug.h"
+#include "logfile.h"
+#include "log.h"
+
+#define dout_context g_ceph_context
+#define dout_prefix *_dout
+#define dout_subsys ceph_subsys_
+
+namespace {
+  seastar::logger& logger() {
+    return crimson::get_logger(ceph_subsys_osd);
+  }
+}  // namespace
+
+struct logfh {
+  std::string log_file_path;
```

Comment:
why do we need keep tracking of the path?

Reply:
i thought it will be usable later on, do you think we can drop it?

Reply:
yeah, if we can check for an empty `log_file_path` and fallback to `cerr`, we don't need store it in this class.